addons/knowledge/views/knowledge_views.xml

```diff
@@ -0,0 +1,247 @@
+                            <!-- Search bar -->
+                            <div class="d-flex align-items-center border-bottom bg-light m-3 o_cursor_pointer">
+                                <div class="flex-grow-1 px-2 py-1">Search an article...</div>
+                                <i class="oi oi-search pr-2" title="search"/>
+                            </div>
+                            <!-- File explorer -->
+                            <div class="flex-grow-1 position-relative">
+                                <div class="o_scroll_view px-3">
+                                    <div class="o_knowledge_tree">
+                                        <div class="p-5 text-center">
+                                            <i class="fa fa-circle-o-notch fa-2x fa-spin" title="loader" role="img"/>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </aside>
+                    <div class="flex-grow-1 position-relative p-0">
+                        <div class="d-flex flex-column h-100">
+                            <!-- Article header -->
+                            <div class="o_knowledge_header d-print-none d-sm-flex flex-row justify-content-between bg-white border-bottom">
+                                <div class="d-flex align-items-center">
+                                    <!-- Breadcrumbs -->
+                                    <ol class="breadcrumb align-items-center p-3" role="navigation">
+                                        <li class="breadcrumb-item d-flex">
+                                            <field attrs="{'readonly': ['|', ('is_locked', '=', True), ('user_can_write', '=', False)]}" class="o_breadcrumb_article_name o_input" name="name"/>
+                                        </li>
+                                    </ol>
+                                    <field name="is_user_favourite" invisible="1"/>
+                                    <i class="fa fa-star o_toggle_favourite" attrs="{'invisible': [('is_user_favourite', '=', False)]}" title="Remove from favourites"/>
+                                    <i class="fa fa-star-o o_toggle_favourite" attrs="{'invisible': [('is_user_favourite', '=', True)]}" title="Add to favourites"/>
+                                    <div attrs="{'invisible': ['|', ('is_locked', '=', False), ('user_can_write', '=', False)]}">
+                                        <i class="fa fa-fw fa-lock" title="locked"/>
+                                    </div>
+                                </div>
+                                <!-- Buttons -->
+                                <div class="d-flex flex-shrink-0 align-items-center pr-2">
+                                    <a type="button" role="button" class="btn btn-secondary btn-create">
```

Comment:
Instead of "btn-create", since it's a custom class that is used (I believe) to generate a handler, I would use something more specific to knowledge (e.g: "o_knowledge_article_form_create").

Same for share and possibly other occurrences.